pdf/internal/cmap/cmap.go

```diff
@@ -182,8 +197,8 @@ func (cmap *CMap) CharcodeToUnicode(code CharCode) (string, bool) {
 	if s, ok := cmap.codeToUnicode[code]; ok {
 		return s, true
 	}
-	common.Log.Debug("ERROR: CharcodeToUnicode could not convert code=0x%04x. cmap=%s. Returning %q",
-		code, cmap, MissingCodeString)
+	// common.Log.Debug("ERROR: CharcodeToUnicode could not convert code=0x%04x. cmap=%s. Returning %q",
```

Comment:
Why comment out?

Reply:
PdfFont.CharcodeBytesToUnicode calls CMap.CharcodeBytesToUnicode . When CMap.CharcodeBytesToUnicode fails, PdfFont.CharcodeBytesToUnicode  falls back to the font encoding and if that fails, it reports an error.
The failure in CMap.CharcodeBytesToUnicode is not necessarily an error and when it is, the error is logged in PdfFont.CharcodeBytesToUnicode